cmd/root.go

```diff
@@ -41,17 +43,26 @@ var root = &cobra.Command{
 		content := string(b)
 		content = strings.ReplaceAll(content, altDelimiter, delimiter)
 		slides := strings.Split(content, delimiter)
+		m, err := meta.New().ParseHeader(slides[0])
+		if err != nil {
+			return errors.New("could not parse theme from header")
```

Comment:
I think instead of `error`ing out we should simply use the default theme

Reply:
Was just thinking that after reading back through. Switching it now.